docs/excel/custom-functions-overview.md

```diff
@@ -28,109 +34,87 @@ In the cloned sample repo, you’ll see the following files:
 
 - **./src/customfunctions.js**, which contains the custom function code (see the simple code example above for the `ADD42` function).
 - **./config/customfunctions.json**, which contains the registration JSON that tells Excel about your custom function. Registration makes your custom functions appear in the list of available functions displayed when a user types in a cell.
-- **./index.html**, which provides a &lt;Script&gt; reference to the JS file. This file does not display UI in Excel.
-- **./manifest.xml**, which tells Excel the location of the HTML, JavaScript, and JSON files; and also specifies a namespace for all the custom functions that are installed with the add-in.
+- **./index.html**, which provides a &lt;Script&gt; reference to the JS file.
+- **./manifest.xml**, which tells Excel the location of the HTML, JavaScript, and JSON files; it also specifies a namespace for all the custom functions that are installed with the add-in.
 
 ### JSON file (./config/customfunctions.json)
 
-The following code in customfunctions.json specifies the metadata for the same `ADD42` function.
+The following code in **customfunctions.json** specifies the metadata for the same `ADD42` function. This metadata includes details on the function's name, description, returned value, parameters, and more. Detailed reference information for the JSON file, including options not used in this example, is at [Custom Functions Registration JSON](custom-functions-json.md).
 
 > [!NOTE]
-> Detailed reference information for the JSON file, including options not used in this example, is at [Custom Functions Registration JSON](custom-functions-json.md).
```

Comment:
I felt like this section was long and confusing so I opted to simplify through comments inline of the code sample. Happy to change back if people want.

Reply:
+1 to Ricky's feedback that comments are not allowed in JSON. Code samples that we provide in the docs (JSON or otherwise) should be valid syntax for the specified language. Suggest that you remove the comments from the JSON code block and instead provide this information after the code block in either a nested list or multiple tables (one table for each object in the JSON:  **function** object (listing its 5 properties), **result** object (listing its 2 properties), **parameter** object (listing its 4 properties).

Reply:
+1 to Kim's suggestion, but put this _above_ the code block or else have a note above the code block telling the reader that more information about the code is below. Reason: readers try to puzzle out code as soon as they reach it. They are seriously annoyed when they discover that their puzzles were answered _after_ the code.